openpype/hosts/maya/plugins/publish/validate_arnold_scene_source_cbid.py

```diff
@@ -0,0 +1,73 @@
+import pyblish.api
+from openpype.hosts.maya.api import lib
+from openpype.pipeline.publish import (
+    ValidateContentsOrder, PublishValidationError, RepairAction
+)
+
+
+class ValidateArnoldSceneSourceCbid(pyblish.api.InstancePlugin):
+    """Validate Arnold Scene Source Cbid.
+
+    It is required for the proxy and content nodes to share the same cbid.
+    """
+
+    order = ValidateContentsOrder
+    hosts = ["maya"]
+    families = ["ass"]
+    label = "Validate Arnold Scene Source CBID"
+    actions = [RepairAction]
+
+    @staticmethod
+    def _get_nodes_data(nodes):
```

Comment:
Method name can be clearer: `get_nodes_by_name`

Reply:
Done